README.md

```diff
@@ -9,9 +9,41 @@
 [![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
 [![js-standard-style](https://img.shields.io/badge/code%20style-standard-brightgreen.svg)](http://standardjs.com/)
 
-This tool allows you to perform a **one way** synchronization from one Contentful space to another.
+This tool allows you to perform a **one way** synchronization of **published** content from one Contentful space to another.
 
-Assuming you have a source space and a destination space, you can keep **published** content synchronized over time. Any draft or unsaved content will not be synchronized.
+The tool makes use of Contentful's [Synchronization API](https://www.contentful.com/developers/docs/concepts/sync/) which means that if you run the tool in the future with the provided token, you will only synchronize new and updated Entries and Assets, as well as remove any that have been deleted.
+
+## What this tool is for
+
+### Development environments
+- You have a Production space where your content editors create and publish content
+- You want your developers to work on new features without touching the production content
+- Use the tool to create copies of your Production space where your Developers can try things out at will
+
+### Field deletion for published Content Types / Entries
+> This feature is only available on Contentful for Content Types which have no Entries. Until the possibility to delete fields comes along, you can use this tool to achieve a similar purpose
+- See the [Deleting Fields](#deleting-fields) section
+
+### Creating new spaces with a similar content model
+- If you want to start out with a content model similar to what you have on another space, you can use the `--content-model-only` option
+
+## What this tool can be used for (but isn't advised)
```

Comment:
There's another section for that below. This one is more like "you can use for this, it works, but it's not advised because of these reasons"